packages/radio-button-plugin/src/packages/RadioGroup.vue

```diff
@@ -33,6 +40,14 @@ export default {
     hasItem() {
       return this.question.choices.length > 0;
     },
+    convertChoices(choices) {
```

Comment:
please consider renaming this method to something more meaningful: `parseChoices`, `createChoices`, etc.